### DISCUSSION TITLE
Appication UI Save Window Size and Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by wiu-wi** [ferraridamiano/ConverterNOW#257] [General]
First, let me thank Damiano Ferrari and everyone involved in creating such an instrumental app. I use the app for various calculations in Ubuntu and I have a few questions and suggestions. 

- How can I save or retain the window size and position on closing the app e.g. as seen in the attachment 
![convert_Now](https://github.com/ferraridamiano/ConverterNOW/assets/106690341/7506d129-dc57-4e65-82d8-05a5bfcb0d59)

-Is it possible to assign keyboard shortcuts for different conversions e,g ctrl + C => calculator, ctrl + A => Area, ctrl + $ => Currencies etc

**Comment by ferraridamiano** [comment]
Hello @wiu-wi!

> How can I save or retain the window size and position on closing the app e.g. as seen in the attachment

While it could be possible to do it on the application side, I don't want to do it for various reason. BUT, if you are using linux you can do it through the window manager. E.g. I know that on KDE (with kwin) you can do it quite easily.

> Is it possible to assign keyboard shortcuts for different conversions e,g ctrl + C => calculator, ctrl + A => Area, ctrl + $ => Currencies

This would be a cool feature to add to the app. Please file an issue (or a PR)
